docs/concepts/how-clerk-works.mdx

```diff
@@ -0,0 +1,111 @@
+
+#### The `__session` cookie
+
+The `__session` cookie is set on the host application that is secured by Clerk (for example `https://dashboard.<example.com>`). It’s a JS cookie, it’s secure, and it contains a short-lived session JWT that lasts 60 seconds. This JWT contains the current session, user, and organization identifiers and must be sent to the application API to authenticate API requests. In SSR, the `__session` cookie travels automatically to the server. ClerkJS ensures that the `__session` cookie is automatically refreshed in Client Side Rendering (CSR) and Server Side Rendering (SSR), ensuring a fresh session is always available.
+
+However, if a `__session` cookie is expired on a request to an application’s backend, the SDK doesn’t know if the session has ended, or if a new short-lived JWT needs to be issued. When an SDK gets into this state, it triggers Clerk's client handshake.
+
+### The client handshake
+
+Clerk uses a client handshake mechanism to resolve a request’s authentication state from unknown (`handshake`) to `signed-in` or `signed-out`. 
+
+With this handshake, Clerk can resolve authentication state on the backend and ensure the request is properly handled as signed in or out, instead of being in a potentially unknown state. The handshake flow relies on redirects to exchange session information between FAPI and the application, ensuring the resolution of unknown authentication states minimizes performance impact and behaves consistently across different framework and language implementations.
+
+Learn more about [how Clerk's client handshake works.](/docs/concepts/client-handshake).
+
+### Client piggybacking
+
+Every Frontend API request returns a response payload that contains two top level keys: 
+
+```json
+{
+   "client": ...,
+   "response": ...
+}
+```
+
+<Callout type="info">
+  The `/v1/client/sessions/:id/tokens` endpoint refreshes the Clerk session tokens and is an exception to this format.
+</Callout>
+
+
+The `response` key contains the resource or resources of each endpoint. The `client` key contains the client piggybacking payload. The client object identifies the current device and contains all sessions, users, and organization data for the current device (see [Main Objects](#main-objects) section). 
+
+This is effectively the necessary frontend state for ClerkJS and powers the JS SDKs, the [React hooks](/docs/references/react/use-user), and [Clerk components](/docs/components/overview). It is a best-effort way for the backend to update the frontend state on every Frontend API request. 
+
+It's not recommended to use Client piggybacking because of the following issues:
+
+1. It’s a very large chunk of a tree-like state and doesn’t change often.
+1. It’s hard to compute on the backend as it requires a lot of data from the database. The computation takes place on every request, adding a significant overhead.
+1. It’s best effort.
+1. It’s not a two-way communication channel.
```

Comment:
This section feels odd here. Who is the audience? Users interested in using the data? Clerk employees? Why are we documenting it if we don't want it to be used?

Reply:
Please move this under the Clerk Frontend API section.

Reply:
@alexisintech ☝️

Reply:
Is this not the section you meant? Because this is where I moved it to:

<img width="820" alt="Screenshot 2024-02-05 at 10 28 17" src="https://github.com/clerk/clerk-docs/assets/98043211/0a3d5990-80cc-4c41-8f6c-eb7998180863">